### PR TITLE
chore: `nx db:migrate:dev` command

### PR DESCRIPTION
**Pull request by overbit**
Close: #7489

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at b89508e</samp>

This pull request updates the commands for running prisma migrations in several packages that use prisma. It adds support for custom database names by using a `name` argument or a default value. 

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-prisma-db/project.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
     "db:clean": {
       "executor": "nx:run-commands",
       "options": {
-        "command": "ts-node scripts/clean.ts",
```
Comment (mulygottlieb) [inline packages/amplication-prisma-db/project.json:81]: Didn't we move away from a custom clean script to a native prisma command?

Comment (overbit) [reply packages/amplication-prisma-db/project.json:81]: That is our current prisma-db (not generated by amplication) and it was pointing to a missing file :S

Comment (overbit) [reply packages/amplication-prisma-db/project.json:81]: Sure, it makes sense. Fixed across the services

**Review by mulygottlieb — APPROVED** [review]

**Comment by mulygottlieb** [comment]
FYI @morhag90 @barshimi - I merged this now (from Dani), please pull from next to get the fix for the prisma:migrate:dev command

**Pull request merged**